mne/channels/layout.py

```diff
@@ -1089,7 +1089,7 @@ def _pair_grad_sensors(
         return picks
 
 
-def _merge_ch_data(data, ch_type, names, method="rms"):
+def _merge_ch_data(data, ch_type, names, method="rms", modality="opm"):
```

Comment:
We've been trying to add these places in our code nowadays
```suggestion
def _merge_ch_data(data, ch_type, names, method="rms", *, modality="opm"):
```